toolkit/primitives/domain/src/lib.rs

```diff
@@ -636,11 +656,20 @@ pub struct BlockProducerRegistration {
 	pub sidechain_pub_key: SidechainPublicKey,
 	pub sidechain_signature: SidechainSignature,
 	pub registration_utxo: UtxoId,
-	pub own_pkh: MainchainAddressHash,
+	// pub own_pkh: MainchainAddressHash,
```

Comment:
Why cannot it be here?

Reply:
If you remember when pair coding we had some trouble with a function and had to pass in own_pkh along with BlockProducerRegistration. When trying to construct a BlockProducerRegistration value for command submission, own_pkh is only available later too. In offchain we don't have this field in BlockProducerRegistration either. So far it seems that own_pkh doesn't belong in BlockProducerRegistration.